server/src/main/java/io/crate/metadata/table/Operation.java

```diff
@@ -64,7 +64,7 @@
     public static final EnumSet<Operation> METADATA_DISABLED_OPERATIONS = EnumSet.of(READ, UPDATE, INSERT, DELETE,
         ALTER_BLOCKS, ALTER_OPEN_CLOSE, ALTER_REROUTE, REFRESH, SHOW_CREATE, OPTIMIZE);
     public static final EnumSet<Operation> LOGICAL_REPLICATED = EnumSet.of(
-        READ, ALTER_BLOCKS, ALTER_REROUTE, OPTIMIZE, REFRESH, COPY_TO, SHOW_CREATE);
+        READ, ALTER_BLOCKS, ALTER_REROUTE, OPTIMIZE, REFRESH, COPY_TO, SHOW_CREATE, ALTER_OPEN_CLOSE);
```

Comment:
This would allow a user to close/open a table with is currently replicating. Is this really needed? We shouldn't allow that (at least not now) as it would technical equal to pausing a subscription as trackers are stopped. If the gap get larger and thus retention leases time out, the shard trackers cannot replicate anymore and we'd have to a) bail out or b) restore the table from scratch using the repository logic.

Reply:
I think initially I added it only because I did ALTER TABLE OPEN/CLOSE manually in tests - but now I recall discussion about hints on failure - quote from slack

```
first step (close tables) fails, the user can safely re-run DROP SUBSCRIPTION.
2nd step (drop setting + subscription) failes, same as ^^
3rd step (open tables) fails, user should be able to manually open the related tables using ALTER TABLE OPEN 
```

for the 3-step failure we need at least let OPEN - for now we don't have such granularity between open/close - I can think of some tricky condition inside `maybeRaiseBlockedException`if a table has LOGICAL_REPLICATED permissions, we need to treat close/open as `only open`

Reply:
Ah right, yes for that we need to allow `open`. Maybe we should replace the operation with dedicated `ALTER_OPEN` + `ALTER_CLOSE` ones?

Reply:
done in https://github.com/crate/crate/pull/12112/commits/faead852afeb7e06b3dafa08994a9fd7438e39f5